docroot/modules/custom/mass_redirects/src/Drush/Commands/MassRedirectsCommands.php

```diff
@@ -31,7 +30,7 @@ public function __construct(
     protected EntityTypeManagerInterface $entityTypeManager,
     protected Connection $connection,
     #[Autowire(service: 'redirect.repository')]
-    protected RedirectRepository $redirectRepository
+    protected RedirectRepository $redirectRepository,
```

Comment:
Reference for anyone, like me, looked at this thinking it is incorrect: https://www.drupal.org/project/coding_standards/issues/3295249